### DISCUSSION TITLE
How to send JSON string to the server in payload after save record in inline editing?

### DISCUSSION THREAD

**Discussion by AlexVDem** [vitmalina/w2ui#2304] [Q&A]
Hi guys. The question about w2ui-1.5.rc1.min.js
When I use form to edit record, push 'Ok' and use "action" in form:
```
actions: {
          "save": function () {
                this.save(function (data) {
                    if (data.status == 'success') {
                        w2ui['records'].reload();
                        $().w2popup('close');
                    }
                });
            },
            "cancel": function () {
                $().w2popup('close');
            },
        }
```
it works like a charm, and my server php script gets payload looks like:
`{"cmd":"save","recid":"1","name":"record_edit","record":{"resp_first_name":"Tiger","resp_last_name":"Wheels","resp_mobile":"+1555123454","resp_phone":"+15551234561","resp_email":"demo@team4work.net","resp_company":"3","resp_department":"1"}}` - this is correct.

But when I use inline editor and use event 'onChange' in the grid:
```
onChange: function(event) {
            this.save(event);
        }
```
I got string like this:
`{"cmd":"save","selected":["1"],"limit":100,"offset":0,"changes":[]}` - this is wrong, payload have no any record changes.

How to send JSON string to the server in payload after save record in inline editing? Is it possible to get whole record with changed field (even if one field changed), and send it in JSON payload like in form editing? Thanks.

**Comment by vitmalina** [comment]
try to wrap it in `event.done(callback)` block, see https://w2ui.com/web/docs/2.0/utils/events

**Comment by AlexVDem** [comment]
Thanks, I'll try to do it.
